### DISCUSSION TITLE
Movement notification

### DISCUSSION THREAD

**Discussion by BooBoss** [JurajNyiri/HomeAssistant-Tapo-Control#640] [Q&A] [answered]
Is there any way to get notifications about movement / people detection by camera. Currently the only override I found is to make automation which enables / disables notification it TAPO app when for example everybody left home. But this gives me notifications on TAPO app only. There's no way to make automations which fires up when movement is detected.

**Comment by JurajNyiri** [comment, marked as answer]
Not at the moment, feature request is opened at https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/issues/304
